ee/clickhouse/queries/breakdown_props.py

```diff
@@ -91,6 +101,22 @@ def get_breakdown_prop_values(
     )[0][0]
 
 
+def _to_value_expression(
+    breakdown_type: Union[Literal["event", "person", "cohort", "group"], None],
+    breakdown: Union[str, List[Union[str, int]], None],
+    breakdown_group_type_index: Optional[int],
+) -> str:
+    if breakdown_type == "person":
+        return get_single_or_multi_property_string_expr(breakdown, "person", "person_props", "value")
+    elif breakdown_type == "group":
+        value_expression, _ = get_property_string_expr(
+            "groups", cast(str, breakdown), "%(key)s", f"group_properties_{breakdown_group_type_index}"
+        )
+        return f"{value_expression} AS value"
+    else:
+        return get_single_or_multi_property_string_expr(breakdown, "events", "properties", "value")
```

Comment:
non-blocking

I get a bit confused by what all the arguments are to these, maybe keyword arguments makes that a little less confusing.